test/e2e/nodenetworkpolicy_test.go

```diff
@@ -891,3 +894,47 @@ func testNodeACNPNestedIPBlockClusterGroupCreateAndUpdate(t *testing.T) {
 	}
 	executeTests(t, testCase)
 }
+
+func testNodeACNPAuditLogging(t *testing.T, data *TestData) {
+	// In a Kind cluster, each Kubernetes Node runs as a Docker container. The iptables LOG target doesn't generate
+	// kernel logs inside containers unless /proc/sys/net/netfilter/nf_log_all_netns is set to 1.  However, this setting
+	// cannot be modified in GitHub Actions.
+	skipIfProviderIs(t, "kind", "The iptables LOG action doesn't generate log in container")
```

Comment:
do you think we should add a note about this in the documentation, in case someone is trying to use the feature on a Kind cluster and cannot figure out why it's not working?

Reply:
Make sense. We should mention it in the documentation.

Reply:
Added.